app/src/main/java/mx/yellowme/youst/common/utils/AnimationHelper.kt

```diff
@@ -0,0 +1,54 @@
+package mx.yellowme.youst.common.utils
+
+import android.animation.Animator
+import com.airbnb.lottie.LottieAnimationView
+
+fun LottieAnimationView.setListener(init: AnimationListenerHelper.() -> Unit) {
+    val listener = AnimationListenerHelper()
+    listener.init()
+    this.addAnimatorListener(listener)
```

Comment:
Este `this` es estrictamente necesario? Podemos removerlo ya estamos en una extension?